yarn.lock

```diff
@@ -9363,6 +9416,11 @@
   resolved "https://registry.yarnpkg.com/@types/json-schema/-/json-schema-7.0.11.tgz#d421b6c527a3037f7c84433fd2c4229e016863d3"
   integrity sha512-wOuvG1SN4Us4rez+tylwwwCV1psiNVOkJeM3AUWUNWg/jDQY2+HE/444y5gc+jBmRqASOm2Oeh5c1axHobwRKQ==
 
+"@types/json-schema@^7.0.7":
+  version "7.0.15"
+  resolved "https://registry.yarnpkg.com/@types/json-schema/-/json-schema-7.0.15.tgz#596a1747233694d50f6ad8a7869fcb6f56cf5841"
+  integrity sha512-5+fP8P8MFNC+AyZCDxrB2pkZFPGzqQWUzpSeuuVLvm8VMcorNYavBqoFcxK8bQz4Qsbn4oUEEem4wDLfcysGHA==
+
```

Comment:
Please consolidate our `@types/json-schema` dependencies, so that we only have a single version installed. You can do this by deleting the two instance here from `yarn.lock` and running the following command:

```
yarn cache clean @types/json-schema && yarn kbn bootstrap
```

This should update all references to use the latest version: `7.0.15`

Reply:
@watson Thanks, I rebased the PR and tried to use the suggested command -- it doesn't seem to affect the lockfile, this entry is not getting removed from it and consolidated with the other entry for this package:

```
"@types/json-schema@^7.0.7":
  version "7.0.15"
  resolved "https://registry.yarnpkg.com/@types/json-schema/-/json-schema-7.0.15.tgz#596a1747233694d50f6ad8a7869fcb6f56cf5841"
  integrity sha512-5+fP8P8MFNC+AyZCDxrB2pkZFPGzqQWUzpSeuuVLvm8VMcorNYavBqoFcxK8bQz4Qsbn4oUEEem4wDLfcysGHA==
```

Running `npx yarn-deduplicate yarn.lock` does the job but introduces more changes to the lockfile that I'd hesitate to merge in this PR.

I'd suggest getting this PR in as is, because it will help us improve the quality of our OpenAPI specs.

I can open a separate one with the lockfile deduplication.

Reply:
> [...] it doesn't seem to affect the lockfile, this entry is not getting removed [...]

You probably forgot to manually edit the lock file with a text editor to remove the duplicate lines before running the suggested command. But glad to see that you found another solution 👍